src/host/srvinit.cpp

```diff
@@ -76,6 +76,14 @@ try
         }
     }
 
+    // Create the accessibility notifier early in the startup process.
+    // We pass along the PTY mode as it won't need to make a notifier then.
+    // The notifiers use expensive legacy MSAA events and the PTY isn't even responsible
+    // for the terminal user interface, so we should set ourselves up to skip all
+    // those notifications and the mathematical calculations required to send those events
+    // for performance reasons.
+    RETURN_IF_FAILED(ServiceLocator::CreateAccessibilityNotifier(args->InConptyMode()));

```

Comment:
I dunno about this. It's less clear (comment aside) that "if this is `true`, `CreateBlah...` does nothing". Why is the decision made in ServiceLocator, rather than ServiceLocator's user?

If ServiceLocator then stored the value and remembered, "I was asked to not create one because i am in a PTY, I will make later decisions based on this" it would make sense... but after all... ServiceLocator's job is to _locate services_ and we would make it easier by simply not giving it services if it isn't supposed to have them.

Reply:
OK I shifted it around.